src/components/Home.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] `ESLint` is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs. Instead of disabling `eslint`, it will be nice to fix the issue.